### PR TITLE
bugfix for __postman_wrapper on init

### PR DESCRIPTION
**Pull request by mikiw**
## Usage related changes

- bugfix for `flush` endpoint call without previous call of `load_l1_messaging_contract`

## Checklist:

- [x] Applied formatting - `./scripts/format.sh`
- [x] No linter errors - `./scripts/lint.sh`
- [x] Performed code self-review
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes
- [x] Linked the issues which this PR resolves
- [x] Updated the tests
- [x] All tests are passing - `./scripts/test.sh`

### CODE REVIEW

File: starknet_devnet/postman_wrapper.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ async def flush(self, state) -> Tuple[dict, list]:
         """Handles all pending L1 <> L2 messages and sends them to the other layer."""
 
         if self.__postman_wrapper is None:
-            return {}
+            return ([], [])
```
Comment (FabijanC) [inline starknet_devnet/postman_wrapper.py:109]: Can be `return [], []` but you can leave it as it is if you think it's clearer that way.

File: starknet_devnet/starknet_wrapper.py [hunk 2]
```diff
@@ -755,24 +755,25 @@ async def mock_message_to_l2(self, transaction: InternalL1Handler) -> dict:
         return transaction.hash_value
 
     async def postman_flush(self) -> dict:
-        """Handles all pending L1 <> L2 messages and sends them to the other layer."""
+        """Handles all pending L1 <> L2 messages if such exists and sends them to the other layer."""
```
Comment (FabijanC) [inline starknet_devnet/starknet_wrapper.py:758]: I don't see these three new words as really necessary. In fact, the current wording is grammatically incorrect.

**Review by FabijanC — CHANGES_REQUESTED** [review]
As discussed: can be a simple test which just attempts flushing before anything has been done previously postman-wise. And we should just assert nothing has failed. Maybe we don't even have to load anything in this new test.

File: starknet_devnet/postman_wrapper.py (hunk 1 above)
Comment (mikiw) [reply starknet_devnet/postman_wrapper.py]: I'll leave it as it is

File: starknet_devnet/starknet_wrapper.py (hunk 2 above)
Comment (mikiw) [reply starknet_devnet/starknet_wrapper.py]: rollbacked in 2eff0d05672cf00b5a63de88b6fc809ce5df205e

**Comment by mikiw** [comment]
I added new test

**Review by FabijanC — APPROVED** [review]

**Pull request merged**